petabvis/dotted_line.py

```diff
@@ -89,13 +90,15 @@ def generate_line(self):
                                                       name=legend_name,
                                                       symbolPen=pg.mkPen("k"),
                                                       symbol=symbol,
-                                                      symbolSize=7)
-                                                      )
+                                                      symbolSize=7))
```

Comment:
pen and size are used again a couple of lines below. It's easier to maintain if the is specified only once. Either as instance variable or in some global config.